PiwikTracker/PiwikUserDefaults.swift

```diff
@@ -58,13 +58,14 @@ internal struct PiwikUserDefaults {
         }
     }
     
-    func clientId() -> String? {
-        return userDefaults.string(forKey: PiwikUserDefaults.Key.visitorID)
-    }
-    
-    func set(clientId: String) {
-        userDefaults.setValue(clientId, forKey: PiwikUserDefaults.Key.visitorID)
-        userDefaults.synchronize()
+    var clientId: String? {
+        get {
+            return userDefaults.string(forKey: PiwikUserDefaults.Key.visitorID)
+        }
+        set {
+            userDefaults.setValue(newValue, forKey: PiwikUserDefaults.Key.visitorID)
+            userDefaults.synchronize()
```

Comment:
Sure. Otherwise we might lose the values we just set. I think we should not risk using the values in case of an early exit (especially an app crash). What do you think @thorstenstark ?